chain/kusama/defaults.go

```diff
@@ -5,20 +5,22 @@ package kusama
 
 import (
 	cfg "github.com/ChainSafe/gossamer/config"
+
+	"github.com/adrg/xdg"
 )
 
 var (
-	// defaultBasePath Default node base directory path
-	defaultBasePath = "~/.gossamer/kusama"
-	// defaultChainSpec is the default chain-spec json path
-	defaultChainSpec = "./chain/kusama/chain-spec-raw.json"
+	// DefaultBasePath default node base directory path
+	DefaultBasePath = xdg.DataHome + "/gossamer/kusama"
+	// DefaultChainSpec is the default chain-spec json path
+	DefaultChainSpec = "./chain/kusama/chain-spec-raw.json"
```

Comment:
Why export these vars? They never get used outside of this package!

Reply:
Good catch. I'm not very fluent in Go; this was on oversight on my end. I'll address this.